tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -1115,11 +947,4 @@ describe(__filename, () => {
       expect(extractId({ match })).toEqual(userId);
     });
   });
-
-  it('dispatches setViewContext when component mounts', () => {
```

Comment:
Why did you delete this test?

Reply:
At the time I thought it was testing some internals that would only make sense with shallow rendering but I see now, not really.  I'll restore it.